unit-tests/src/test/scala/scala/scalanative/unsafe/CArrayBoxingTest.scala

```diff
@@ -8,16 +8,16 @@ import scalanative.junit.utils.AssertThrows._
 
 import scalanative.unsafe.Nat._
 import scalanative.unsigned._
-import scalanative.libc.stdlib.malloc
+import scalanative.libc.stdlib.calloc
 import java.lang.Long.toHexString
 
 class CArrayBoxingTest {
   var any: Any = null
 
   @noinline lazy val nullArr: CArray[Byte, _4] = null
-  @noinline lazy val arr: CArray[Byte, _4] = !malloc(64.toULong)
+  @noinline lazy val arr: CArray[Byte, _4] = !calloc(1.toUInt, 64.toUInt)
```

Comment:
What is the advantage of clearing the memory.  On a quick read, arr & arr2 are used as addresses and the contents not considered before being set (if ever, I did not trace that).

Have I missed something?